src/java/io/bazel/rulesscala/scalac/ScalaCInvoker.java

```diff
@@ -255,6 +255,37 @@ private static void processRequest(List<String> args) throws Exception {
           reporter.flush();
           throw new RuntimeException("Build failed");
       } else {
+        /**
+         * See if there are java sources to compile
+         */
+        if (ops.javaFiles.length > 0) {
+          StringBuilder cmd = new StringBuilder();
+          cmd.append(ops.javacPath);
+          if (ops.jvmFlags != "") cmd.append(ops.jvmFlags);
```

Comment:
should these not be separate args to the Process builder rather than a string concat?